### PR TITLE
Integration test to measure max NFTs mintable in a single proposal.

### PR DESCRIPTION
**Pull request by 0xekez**
based on tuning on my local setup, the current number is 55.

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -63,3 +63,14 @@ workspace-optimize:
 		--mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
 		--platform linux/amd64 \
 		cosmwasm/workspace-optimizer:0.12.11
+
+workspace-optimize-arm:
```
Comment (JakeHartnell) [inline justfile:67]: :heart:

**Review by JakeHartnell — APPROVED** [review]
LGTM.

File: justfile [hunk 2]
```diff
@@ -63,3 +63,14 @@ workspace-optimize:
...
+    docker run --rm -v "$(pwd)":/code \
+        --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
+        --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+        --platform linux/arm64 \
+        cosmwasm/workspace-optimizer-arm64:0.12.11
+    # rename the wasm files so that they work with cosm-orc
+    # integration test contract names (which are expected to be the
+    # contract name, without the aarch64 postfix).
+    for file in artifacts/*-aarch64.wasm; do mv -- "$file" "${file%-aarch64.wasm}.wasm"; done
```
Comment (de-husk) [inline justfile:76]: I tried to handle this situation here:  https://github.com/de-husk/cosm-orc/blob/main/src/orchestrator/cosm_orc.rs#L153-L157

I wonder why it's not working in this case?

File: ci/integration-tests/src/helpers/helper.rs [hunk 3]
```diff
@@ -97,12 +100,194 @@ pub fn create_dao(
...
+	.orc
+	.query(
+	    "dao_proposal_single",
+	    &dao_proposal_single::msg::QueryMsg::ProposalCreationPolicy {}
+	).unwrap()
+	.data()
+	.unwrap()
+    else {
+	panic!("expected pre-propose module")
+    };
```
Comment (de-husk) [inline ci/integration-tests/src/helpers/helper.rs:130]: I like this pattern

**Review by de-husk — APPROVED** [review]

File: justfile (hunk 2 above)
Comment (0xekez) [reply justfile:76]: possible it was failing for unrelated reasons when i added this. i'll test before merging.

**Review by bekauz — APPROVED** [review]

File: justfile (hunk 2 above)
Comment (0xekez) [reply justfile:76]: @de-husk confirmed that your solution works and my change was not needed. thanks for pointing this out!

**Pull request merged**